add glide support, xml lib, fixing button touch up on parent, add cancel in onscreenkeyboard, new component : slider, grid, (combobox + list)

mifmasterz opened the pull request:
before accept this pull request, maybe you want to test it from this repo : https://github.com/Gravicode/GlideX 

some homework:
1. need to change the naming convention in order to comply with the custom solution naming conventions
2. review the glide source code
3. need more tests
4. delete unused comments 
5. need to repair image quality for onscreen keyboard..

mifmasterz committed:
delete some comments

mifmasterz committed:
remove test app

gus-ghielec commented:
There are too many changes that needs to be broken down. We should not mix fixing changes with adding new features. Also XML should not be part of this work. You can send a PR that fixes UI issues. We are going to be reviewing/adding XML as a separate nuget, which you can help with #570 As far as adding glide back, we need to review, and if approved, needs to be documented.

Palomino34 commented:
Also, please do some change:
- "AssemblyInfo.cs" need to be same format to what we have.
- DT_TruncateAtBottom, DT_AlignmentCenter..... already in Drawing, no need make new Bitmap class.
- Use correct namespace: No longer "namespace GHI.xxxx"
- FontManager, Font, Image resource... Doesn't UI has them already? Can we use just one? When use GlideX, it will load UI? If so it will load double resource which is too larage.
- Packages.config need to be same as other ones.
- GHIElectronics.TinyCLR.GlideX.csproj need to be same as other ones.
- Where is *.nuspec file?

And as Gus said, we need separately PR for each issues.

mifmasterz committed:
"AssemblyInfo.cs" need to be same format to what we have. ok
DT_TruncateAtBottom, DT_AlignmentCenter..... already in Drawing, no need make new Bitmap class. ok
Use correct namespace: No longer "namespace GHI.xxxx" ok
FontManager, Font, Image resource... Doesn't UI has them already? Can we use just one? When use GlideX, it will load UI? If so it will load double resource which is too larage. -> GlideX is only loader library, it will load "old glide xml" to new UI library (GHIElectronics.TinyCLR.UI / WPF), yes it will use UI to render canvas, so no duplicate resources
Packages.config need to be same as other ones. ok
GHIElectronics.TinyCLR.GlideX.csproj need to be same as other ones ok
Where is *.nuspec file? ok added

mifmasterz commented:
"AssemblyInfo.cs" need to be same format to what we have. **ok**
DT_TruncateAtBottom, DT_AlignmentCenter..... already in Drawing, no need make new Bitmap class. **ok**
Use correct namespace: No longer "namespace GHI.xxxx" **ok**
FontManager, Font, Image resource... Doesn't UI has them already? Can we use just one? When use GlideX, it will load UI? If so it will load double resource which is too larage. -> GlideX is only loader library, it will load "old glide xml" to new UI library (GHIElectronics.TinyCLR.UI / WPF), yes it will use UI to render canvas, so no duplicate resources
Packages.config need to be same as other ones. **ok**
GHIElectronics.TinyCLR.GlideX.csproj need to be same as other ones **ok**
Where is *.nuspec file? **ok added**
